CHANGELOG/CHANGELOG-1.13.md

```diff
@@ -0,0 +1,89 @@
+# Changelog 1.13
+
+## 1.13.0 - 2023-07-24
+
+### Added
+
+- Add AdminNetworkPolicy support in Antrea to align with K8s NetworkPolicy API, and document the introduction and usage. ([#5170](https://github.com/antrea-io/antrea/pull/5170) [#5270](https://github.com/antrea-io/antrea/pull/5270), [@Dyanngg])
+- Support DSR mode for Service's external addresses including LoadBalancerIPs and ExternalIPs on AntreaProxy. ([#5202](https://github.com/antrea-io/antrea/pull/5202) [#5251](https://github.com/antrea-io/antrea/pull/5251), [@tnqn])
+- Containerize Windows userspace OVS processes and run them in a container of the Antrea Agent Pod to align with the Linux design. ([#4936](https://github.com/antrea-io/antrea/pull/4936) [#5052](https://github.com/antrea-io/antrea/pull/5052) [#5071](https://github.com/antrea-io/antrea/pull/5071), [@rajnkamr] [@Atish-iaf] [@NamanAg30])
```

Comment:
#5071 is for adding containerd support in PrepareNode script. 

From PR #5071 summary 
> This patch gives the user the option of using containerd runtime while using the PrepareNode.ps1 for configuring k8s on windows node.

Reply:
Add a new item for this PR.